### DISCUSSION TITLE
is it possible to have a calender that does not sync with Google?

### DISCUSSION THREAD

**Discussion by nutpantz** [FossifyOrg/General-Discussion#477] [Q&A]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Calendar/issues) or [closed](https://github.com/FossifyOrg/Calendar/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Calendar/discussions) or [closed](https://github.com/FossifyOrg/Calendar/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Calendar/blob/master/CONTRIBUTING.md).
- [X] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

is it possible to have a calender that does not sync with Google?
(As i under stand it androids calender is forced synced with google)

### Why do you want this feature?

So that my appointments do not end up on Google or any ones else's servers

**Comment by Aga-C** [comment]
If you don't add an event to CalDAV synchronized calendar, it's only in this app. Other apps have no access to it.
